packages/web/src/components/app-redirect-popover/components/AppRedirectPopover.tsx

```diff
@@ -119,7 +119,8 @@ const AppRedirectPopover = ({
   const onClick = () => {
     onBeforeClickApp()
     const pathname = getPathname()
-    const newHref = `https://redirect.audius.co${APP_REDIRECT}${pathname}`
+    const newHref = `audius://${pathname}`
```

Comment:
nice. this redirect.audius.co thing is super broken.
does this work if the app is not installed?

Reply:
No this doesn't work when the app isn't installed, I assume that's why we had the redirect thing?